client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.ts

```diff
@@ -64,7 +64,7 @@ export class EditCustomConfigComponent extends FormReactive implements OnInit {
   }
 
   ngOnInit () {
-    const formGroupData = {
+    const formGroupData: any = {
```

Comment:
`formGroupData` is not an any type, it's defined by the assignment

Reply:
what do you mean? which type should I make it?
without any, I think `noImplicitAny` won't work

Reply:
>src/app/+admin/config/edit-custom-config/edit-custom-config.component.ts(78,7): error TS7018: Object literal's property 'signupEnabled' implicitly has an 'any' type.

this error gets thrown for all elements of `formGroudData` where `: null`